PUSH_INSTRUCTIONS.md

```
@@ -0,0 +1,99 @@
+> Task :classes
+> Task :compileTestJava
+> Task :processTestResources
+> Task :testClasses
+> Task :test
+
+7 files changed, 219 insertions(+), 30 deletions(-)
+```
+
+## Commit Details
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)